### PR TITLE
[docs] Add documentation for executable steps

### PR DESCRIPTION
**Pull request by draker94**

### CODE REVIEW

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 1]
```diff
@@ -261,3 +261,85 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
...
+When the condition `$condition` is true I do$stepsToExecute
+----
+
+Alias:
+[source,gherkin]
+----
+When the condition '$condition' is true I do$stepsToExecute
+----
+
+* `$condition` - The verifiable condition.
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:309]: Please add the description how is result of the condition checked: true vs anything to else

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 2]
```diff
@@ -261,3 +261,85 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
...
+* `$stepsToExecute` - The ExamplesTable with a single column containing the steps to execute if condition is true.
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:310]: it makes sense to define the name of the single column

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 3]
```diff
@@ -261,3 +261,85 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
...
+.Click on the element if parent element is found
+[source,gherkin]
+----
+When I save number of elements located `By.xpath(//*[@class='outerElement'])` to SCENARIO variable `numberOfOuterElements`
+When the condition `#{eval(${numberOfOuterElements} == 1)}` is true I do
+|step                                                                   |
+|When I click on element located `By.xpath(//*[@class='innerElement'])` |
+----
+
+=== Execute steps when a variable is not set
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:321]:
```suggestion
=== Execute steps if a variable is not set
```

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 4]
```diff
@@ -261,3 +261,85 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
...
+
+Execute steps if the variable with `$name` is not set into context.
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:323]: Please see the above comment about step intro and parameters

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 5]
```diff
@@ -261,3 +261,86 @@ When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'`
 When I execute SQL query `SELECT * FROM test.launch_rockets WHERE name='Proton'` against `prod` and save result to scenario variable `prod-data`
 Then `${preprod-data}` is = `${prod-data}`
 ----
+
+=== Execute steps N times
```
Comment (ikalinin1) [inline docs/modules/commons/pages/vividus-steps.adoc:265]: Please add an example of usage nested step inside nested steps

Comment (ikalinin1) [reply docs/modules/commons/pages/vividus-steps.adoc:265]: For all the steps

**Review by uarlouski — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**